### PR TITLE
Add "audit-remote-libvirt" into openQA

### PR DESCRIPTION
**Pull request by lilyeyes**
Add "audit-remote-libvirt" of "audit-test" into openQA

- Related ticket: https://progress.opensuse.org/issues/96531
- Needles: NA
- MR: https://gitlab.suse.de/security/audit-test-sle15/-/merge_requests/22
- Verification run: https://openqa.suse.de/tests/6820236 (pass)

### CODE REVIEW

File: lib/audit_test.pm [hunk 1]
```diff
@@ -56,7 +56,17 @@ sub run_testcase {
     # Export MODE
     assert_script_run("export MODE=$audit_test::mode");
 
-    assert_script_run('./run.bash', timeout => $args{timeout});
+    # Test case 'audit-remote-libvirt' does not generate any logs
+    if ($testcase eq 'audit-remote-libvirt') {
+        # Note: the outputs of run.bash can not be saved to "./$file" so saved to "../$file"
```
Comment (rfan1) [inline lib/audit_test.pm:61]: change "so saved to"    -> ", so save to"

File: tests/security/cc/audit_remote_libvirt.pm [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+# Copyright © 2021 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved. This file is offered as-is,
+# without any warranty.
+#
+# Summary: Run 'audit-remote-libvirt' test case of 'audit-test' test suite
+# Maintainer: llzhao <llzhao@suse.com>
+# Tags: poo#
```
Comment (rfan1) [inline tests/security/cc/audit_remote_libvirt.pm:12]: missed the poo number here, right?

File: lib/audit_test.pm (hunk 1 above)
Comment (lilyeyes) [reply lib/audit_test.pm]: > change "so saved to" -> ", so save to"

done.

File: tests/security/cc/audit_remote_libvirt.pm (hunk 2 above)
Comment (lilyeyes) [reply tests/security/cc/audit_remote_libvirt.pm]: Good finding. done.

File: tests/security/cc/audit_remote_libvirt.pm [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+    zypper_call("in qemu libvirt virt-install virt-manager");
+
+    # Start libvirtd daemon and start the default libvirt network
+    assert_script_run("systemctl start libvirtd");
+    assert_script_run("virsh net-start default");
+    assert_script_run("systemctl is-active libvirtd");
+    assert_script_run("virsh net-list | grep default | grep active");
+
+    # Download the pre-installed guest images and sample xml files
+    my $vm_name      = 'vm-swtpm-legacy';
```
Comment (Amrysliu) [inline tests/security/cc/audit_remote_libvirt.pm:37]: IMHO, If the `$vm_name` could change to something about `audit-remote`  will be better.

Comment (lilyeyes) [reply tests/security/cc/audit_remote_libvirt.pm:37]: > IMHO, If the `$vm_name` could change to something about `audit-remote` will be better.

You are right, I have thought this but I reused test case 'swtpm' data files if I change the vm name I have to change the data files' contents accordingly (the code change is not so simple).
So I decided not to do that.

**Review by rfan1 — APPROVED** [review]
LGTM

**Review by Amrysliu — COMMENTED** [review]
LGTM

**Review by jouyingbin — APPROVED** [review]
LGTM,thanks.

**Review by Amrysliu — APPROVED** [review]

**Pull request merged**